model01/Makefile

```diff
@@ -6,16 +6,20 @@ deps:
 		cd $(HOME)/Arduino/hardware/keyboardio/avr && make update-submodules; \
 	else \
 		curl https://downloads.arduino.cc/arduino-1.8.5-linux64.tar.xz -o arduino.tar.xz && \
-		tar -xvf arduino.tar.xz -C arduino && \
+		mkdir arduino && \
+		tar -xvf arduino.tar.xz && \
 		rm arduino.tar.xz && \
 		sudo curl https://raw.githubusercontent.com/keyboardio/Kaleidoscope/master/etc/99-kaleidoscope.rules -o /etc/udev/rules.d/99-kaleidoscope.rules && \
 		git clone git@github.com:keyboardio/Model01-Firmware.git && \
 		mkdir -p $(HOME)/Arduino/hardware/keyboardio && \
 		git clone --recursive git@github.com:keyboardio/Arduino-Boards.git $(HOME)/Arduino/hardware/keyboardio/avr; \
 	fi
 
+permissions:
+	sudo usermod -a -G uucp $(USER) && \
+	sudo usermod -a -G lock $(USER)
+
 flash:
 	cp ./Model01-Firmware.ino ./Model01-Firmware && \
-	sudo usermod -a -G uucp $(USER) && \
-	sudo usermod -a -G lock $(USER) && \
-	su - $(USER) -c "bash -c \"cd $(shell pwd)/Model01-Firmware && ARDUINO_PATH=../arduino make flash\""
+	cd $(shell pwd)/Model01-Firmware && \
+	ARDUINO_PATH=../arduino-1.8.5 make flash
```

Comment:
Probably want this to be `../arduino` and then make sure the `tar` call actually dumps the output there. Otherwise if we download a newer version of Arduino the directory name will be different.